.github/workflows/release.yml

```
@@ -0,0 +1,48 @@
+name: Release
+
+on:
+  release:
+    types:
+      - created
+
+permissions:
+  contents: read
+
+jobs:
+  push:
+    if: github.repository == 'mime-types/mime-types-data'
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    
    env:
      rubygems_release_gem: true
```

I’m going to be updating my meta-plugin `halostatue/hoe-halostatue`  to *not* load `doofus` and `git2` if `rubygems_release_gem` is set to `true`.